Makefile

```diff
@@ -23,10 +23,57 @@ $(MAKEFILE):
 
 -include $(MAKEFILE)
 
+# Conf vars
+GITHUB_API_KEY ?=
+LOGS_PATH ?= /var/log/docsrv
+LANDING_PATH ?= $(SHARED_PATH)/landing
+INIT_PATH ?= $(shell pwd)/../ci/docs/init.d
+CONF_PATH ?= $(shell pwd)/../ci/docs/conf.d
+
+#vars
+docker_container_name = docsrv-instance
+docker_image_name = docsrv-image
+
 #tools
 mkdir := mkdir -p
 goBuild := CGO_ENABLED=0 go build -o
+remove := rm -rf
```

Comment:
From the [gnu.org/make/manual](https://www.gnu.org/software/make/manual/html_node/Using-Variables.html)
> _It is traditional to use upper case letters in variable names, but **we recommend using lower case letters for variable names that serve internal purposes** in the makefile, and reserving upper case for parameters that control implicit rules or for parameters that the user should override with command options_
Considering that, those vars should be in lowercase since the user should not configure them.

But there are other examples with the opposite...
so... should I use lowercase as recommended by gnu.org, or should I use uppercase as can be seen in some places?